backend/cmd/cmd.go

```diff
@@ -29,6 +30,7 @@ type ServiceConfig struct {
 	KeyGenBufferSize     int
 	KgsHostname          string
 	KgsPort              int
+	AuthTokenLifetime    time.Duration
```

Comment:
FYI, in the future, we may want to consider eliminating this repetitive struct.

Reply:
I was thinking it too, the config struct is becoming too big to manually initialize!